watertap/data/techno_economic/buffer_tank.yaml

```diff
@@ -28,3 +28,8 @@ default:
   default_removal_frac_mass_solute:
     value: 0
     units: dimensionless
+  removal_frac_mass_solute:
```

Comment:
this model is just a tank without any removal, so you should remove this

Reply:
Removed.